week-1-practise/level-2/2-format-currency.js

```diff
@@ -6,7 +6,10 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(price) {
+  // currency formatting
```

Comment:
Good result!
In terms of comments, it's good to remember that even though they might seem useful and descriptive, we should aim to make our code - that is, function and variable names - clear and descriptive enough to minimise the need for comments :) I think in this case the function name `formatCurrency` is meaningful enough

Reply:
Thanks  martynamaron .I got what do you mean